plugins/tekton/src/utils/tekton-utils.ts

```diff
@@ -152,6 +159,29 @@ export const getDuration = (seconds: number, long?: boolean): string => {
   return duration.trim();
 };
 
+export const descendingComparator = (
```

Comment:
good to have tests for these utils as well

Reply:
Added tests